docs/using/backup.md

```diff
@@ -57,46 +46,75 @@
 
     - *Name*:
     - *Description*:
-    - *Type:*
-        - *S3*:
+    - *Type*: Choose the type of storage.
+        - *S3*: Use [Amazon AWS S3]
+            - *Endpoint*: The S3 backup location endpoint.
+            - *Bucket Name*:
+            - *Access Key*:
+            - *Secret Key*:
         - *Local Client:* (Not currently implemented)
         - *Local Server:* (Not currently implemented)
-    - *Endpoint:*
-    - *Bucket Name:*
-    - *Access Key:*
-    - *Secret Key:*
 
 4. Click *Add* to add the location or *Test* to test the connection.
 
-## Creating a backup
+## MySQL backup preconditions
 
-!!! hint alert alert-success "Tip"
-    You must [add a backup storage location](#adding-a-storage-location) before backing up.
+- There is only one MySQL instance running on the node.
```

Comment:
I think we also should mention that `pmm-agent` should run on the same node with mySQL.

Reply:
```suggestion
- [PMM Client](../setting-up/client/index.md) is installed and running on the node.

- There is only one MySQL instance running on the node.
```

Reply:
Nice 👍